src/openrct2-win/openrct2-win.cpp

```diff
@@ -40,6 +40,7 @@ int wmain(int argc, wchar_t** argvW, [[maybe_unused]] wchar_t* envp)
     std::vector<const char*> argv;
     std::transform(
         argvStrings.begin(), argvStrings.end(), std::back_inserter(argv), [](const auto& string) { return string.c_str(); });
+    argv.push_back(nullptr); // Ensure that argv[argc] == nullptr, as mandated by the standard
```

Comment:
Thanks, but nitpicky: per our coding style, we (almost) always place comments on a separate line before the line itself.